apps/gateway/generate-schema.sh

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/env bash
+
+mkdir -p schemas
+
+cat ../accounts/internal/app/graph/*.graphqls >  ./schemas/accounts-api.schema
+cat ../accounts/internal/app/graph/struct-to-graphql/*.graphqls >>  ./schemas/accounts-api.schema
+
+cat ../console/internal/app/graph/*.graphqls >  ./schemas/console-api.schema
+cat ../console/internal/app/graph/struct-to-graphql/*.graphqls >>  ./schemas/console-api.schema
+
+cat ../container-registry/internal/app/graph/*.graphqls >  ./schemas/container-registry-api.schema
+cat ../container-registry/internal/app/graph/struct-to-graphql/*.graphqls >>  ./schemas/container-registry-api.schema
+
+cat ../infra/internal/app/graph/*.graphqls >  ./schemas/infra-api.schema
+cat ../infra/internal/app/graph/struct-to-graphql/*.graphqls >>  ./schemas/infra-api.schema
+
+cat ../auth/internal/app/graph/*.graphqls >  ./schemas/auth-api.schema
+cat ../message-office/internal/app/graph/*.graphqls >  ./schemas/message-office-api.schema
+
+rover supergraph compose --config ./supergraph.yml --output prod-schema.graphql --elv2-license accept
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Explicit acceptance of license in automation scripts.

While this ensures the script runs without interactive input, consider the implications of automatically accepting licenses in your CI/CD pipelines.